app/javascript/utilities/connect/getUnopenedChannels.jsx

```diff
@@ -23,9 +23,9 @@ class UnopenedChannelNotice extends Component {
 
   componentDidMount() {
     const { pusherKey } = this.props;
-    const appName = document.body.dataset.appName;
+    const appDomain = document.body.dataset.appDomain;
```

Comment:
refactor (non-blocking): Prefer destructuring.

```suggestion
    const { appDomain } = document.body.dataset;
```